### PR TITLE
[ALLUXIO-2385]Improve precondition check in JournalReader.java

### PR DESCRIPTION
**Pull request by haoxiang47**
https://alluxio.atlassian.net/browse/ALLUXIO-2385

### CODE REVIEW

File: core/server/src/main/java/alluxio/master/journal/JournalReader.java [hunk 1]
```diff
@@ -55,7 +55,7 @@
    * @param journal the handle to the journal
    */
   JournalReader(Journal journal) {
-    mJournal = Preconditions.checkNotNull(journal);
+    mJournal = Preconditions.checkNotNull(journal,"journal");
```
Comment (HorizonNet) [inline core/server/src/main/java/alluxio/master/journal/JournalReader.java]: There should be a white space after the comma.

File: core/server/src/main/java/alluxio/worker/file/FileSystemWorkerClientServiceHandler.java [hunk 2]
```diff
@@ -49,7 +49,7 @@
    * @param worker the file system worker which will handle most of the requests
    */
   public FileSystemWorkerClientServiceHandler(FileSystemWorker worker) {
-    mWorker = Preconditions.checkNotNull(worker);
+    mWorker = Preconditions.checkNotNull(worker,"worker");
```
Comment (HorizonNet) [inline core/server/src/main/java/alluxio/worker/file/FileSystemWorkerClientServiceHandler.java]: Dito

**Comment by HorizonNet** [comment]
@haoxiang47 I left two minor comments. Could you also add the email address used in one of your commits to your GitHub account (see the comment above). After changing this I will give a LGTM.

File: core/server/src/main/java/alluxio/worker/file/FileSystemWorkerClientServiceHandler.java (hunk 2 above)
Comment (haoxiang47) [inline core/server/src/main/java/alluxio/worker/file/FileSystemWorkerClientServiceHandler.java]: hi @HorizonNet I have finished to add whitespce and email address

**Comment by HorizonNet** [comment]
alluxio-bot, check this please

**Comment by haoxiang47** [comment]
alluxio-bot, check this please

**Comment by haoxiang47** [comment]
alluxio-bot, check this please

**Comment by haoxiang47** [comment]
alluxio-bot, check this please

**Comment by haoxiang47** [comment]
alluxio-bot, check this please

**Comment by haoxiang47** [comment]
@HorizonNet sorry HorizonNet, I change my commit email address, I saw git log it has changed,
`
commit 782347cc502ba2eeed3a764a7f710323d825bb72
Author: linhaoxiang <linhaoxiang32@gmail.com>
commit 782347cc502ba2eeed3a764a7f710323d825bb72
Author: linhaoxiang <linhaoxiang32@gmail.com>
Date:   Sun Oct 30 02:37:26 2016 +0800

    add whitespaces and add email address
`
why it can't pass the check?

**Comment by HorizonNet** [comment]
The problem is in your [first commit](https://github.com/Alluxio/alluxio/pull/4278/commits/e55351ec93eca7187d8d32c1844c04da7f48e600). Did you [follow the instructions](https://help.github.com/articles/adding-an-email-address-to-your-github-account/) provided above?

**Comment by haoxiang47** [comment]
alluxio-bot, check this please

**Comment by haoxiang47** [comment]
@HorizonNet hi HorizonNet, I have passed checks, please give some feedback

**Comment by HorizonNet** [comment]
LGTM

**Pull request merged**